blatann/services/nordic_uart/service.py

```diff
@@ -71,13 +71,16 @@ def add_to_database(cls, gatts_database, max_characteristic_size=None):
         return cls(service, max_characteristic_size)
 
 
-class NordicUartClient(object):
+class NordicUartClient(object, max_characteristic_size=None):

```

Comment:
This parameter should be part of `__init__`, not the class

Reply:
Yes of course, I have made a new commit